modules/navier_stokes/src/postprocessors/VolumetricFlowRate.C

```diff
@@ -57,7 +58,9 @@ VolumetricFlowRate::VolumetricFlowRate(const InputParameters & parameters)
     _rc_uo(isParamValid("rhie_chow_user_object")
                ? &getUserObject<RhieChowFaceFluxProvider>("rhie_chow_user_object")
                : nullptr),
-    _subtract_mesh_velocity(getParam<bool>("subtract_mesh_velocity"))
+    _subtract_mesh_velocity(isParamValid("subtract_mesh_velocity")
+                                ? getParam<bool>("subtract_mesh_velocity")
+                                : _fe_problem.haveDisplaced())
```

Comment:
I think this should actually be an error if we don't have mesh displacements and someone wants subtract mesh velocity.

Reply:
I mean we will subtract 0, but I think we should expect the user to have this right.